traffic_portal/app/src/common/modules/table/serverCapabilityServers/table.serverCapabilityServers.tpl.html

```diff
@@ -20,11 +20,12 @@
 <div class="x_panel">
     <div class="x_title">
         <ol class="breadcrumb pull-left">
-            <li><a href="#!/server-capabilities">Server Capabilities</a></li>
-            <li><a ng-href="#!/server-capabilities/{{serverCapability.name}}">{{::serverCapability.name}}</a></li>
+            <li><a ng-click="navigateToPath('/server-capabilities')">Server Capabilities</a></li>
+            <li><a ng-click="navigateToPath('/server-capabilities/edit?name=' + serverCapability.name)">{{::serverCapability.name}}</a></li>
```

Comment:
Anchor elements should not have `click` event handlers. They should use their `href` properties (or `ng-href` as appropriate) so that they properly act as links.

Reply:
@ocket8888: I am confused, a lot of pages (co-ordiantes, deliveryservice etc), which are referencing breadcrumbs have ng-click. But your comment says otherwise. So, why the discrepancy?

Reply:
Also, I broke the bread-crumb fix in its own PR.

Reply:
> a lot of pages (co-ordiantes, deliveryservice etc), which are referencing breadcrumbs have ng-click.

Those pages are doing that wrong. They should also not be using click events for navigation.

Reply:
Do we need to create an issue to refactor incorrect use of ng-click?

Reply:
No, I've added it to #3594. It's likely that'll just never get fixed and we'll just have to do it right in TPv2.